apps/web/playwright/auth/auth-index.e2e.ts

```diff
@@ -55,7 +55,6 @@ test.describe("Can signup from a team invite", async () => {
     await newPage.fill('input[name="username"]', proUser.username); // Invalid username
     await newPage.fill('input[name="password"]', testUser.password);
     await submitAndWaitForResponse(newPage, "/api/auth/signup", { expectedStatusCode: 409 });
-    await expect(newPage.locator('text="Username or email is already taken"')).toBeVisible();
```

Comment:
Why were these removed @hbjORbj ?

Aren't these ensuring proper i18n funcationality?

Reply:
Oh it was unintentional. I just added it back